src/main/java/com/currencyfair/onesignal/model/player/ViewDeviceResponse.java

```diff
@@ -22,13 +22,25 @@
  * Response class for {@link com.currencyfair.onesignal.OneSignal#viewDevice(String, String, String)} request.
  */
 public class ViewDeviceResponse extends Device {
+    /**
+     * The device's OneSignal ID
+     */
+    private String id;
```

Comment:
Not sure is that id need `@JsonProperty` as well ?

Reply:
I guess it is optional if the property name is the same as that of variable, for example https://github.com/CurrencyFair/OneSignal-Java-SDK/blob/2393a98adae4cc014e7e7033d00473b9ade16940/src/main/java/com/currencyfair/onesignal/model/player/ViewDevicesResponse.java#L37